src/shared/database/index.ts

```diff
@@ -431,9 +432,10 @@ export async function getAgreementScores(
 export async function signingToMaster(
   signing_key: string,
 ): Promise<string | undefined> {
-  return query('manifests')
+  return query('validators')
     .select('master_key')
     .where({ signing_key })
+    .where('revoked', '=', 'false')
```

Comment:
This change means that validators that haven't updated their signing key yet, but have a master key, will be scored based on their master key. The `revoked = false` is an additional check so old revoked signing keys won't be under the same bucket.